README.md

```diff
@@ -178,6 +178,15 @@ gesdinet_jwt_refresh_token:
     firewall: api
 ```
 
+### Config Refresh token parameter Name
+
+You can define refresh token parameter name. Default value is refresh_token. You can change this value adding this line to your config.yml file:
```

Comment:
config.yml is only for Sf 3.x version, please update this. Something like "to your config"